frontend/src/pages/_dashboard/:slug/project/SubmissionForm.js

```diff
@@ -283,8 +295,8 @@ export default () => {
                             name="demo"
                             render={({ field, form }) => (
                                 <FormControl
-                                    label="Demo"
-                                    hint="Download your presentation video of the project to Vimeo and link it here. Max duration is 2 minutes. (Make sure everyone receiving the link has access to the video.) If you have any materials, such as a presentation deck or a demo, they should be presented in the video."
+                                    label="Demo URL or Coupon Code"
```

Comment:
What is coupon code referring to?

We also had to hard code some changes to our last event, since we wanted video submissions. 

The demo field is good as it is, but maybe the label and hint in the submission form should be something we can configure on event by event basis (https://github.com/hackjunction/hackplatform/issues/107#issue-589843835)

Reply:
It is also a necessary hardcoding for the 'capture-the-flag' style online events we plan to hold in the coming months. It would be great if this label could be customizable!